app/views/programming_groups/new.html.slim

```diff
@@ -5,18 +5,22 @@ h1 = t('programming_groups.new.create_programming_pair')
       => t('programming_groups.new.own_user_id')
       b
         = current_user.id_with_type
-    p
+
+    button.btn.btn-success data-bs-toggle= 'modal' data-bs-target= '#modal-info-pair-programming'
+      i.fa-solid.fa-circle-info
+      = t('shared.pair_programming_info')
+
+    p.mt-4
       = t('programming_groups.new.enter_partner_id', exercise_title: @exercise.title)
     = render('form')
 
     div.mt-4
-      a.btn.btn-success href=new_exercise_programming_group_path(@exercise) == t('programming_groups.new.check_invitation')
-
-      p.mt-4
-        == t('programming_groups.new.work_alone', path: implement_exercise_path(@exercise))
+      == t('programming_groups.new.work_alone', path: implement_exercise_path(@exercise))
 
   .col-md-6
     h5 = t('programming_groups.new.find_partner_title')
     p
       = t('programming_groups.new.find_partner_description')
     iframe name="embed_readwrite" src="https://etherpad.xopic.de/p/find_programming_group_for_exercise_#{@exercise.id}?showControls=true&showChat=true&showLineNumbers=true&useMonospaceFont=false" width="100%" height="300" style='border: 1px solid black;'
+
+= render('shared/modal', classes: 'modal-lg', id: 'modal-info-pair-programming', template: 'application/info_pair_programming', title: "Pair Programming Info")
```

Comment:
Potentially, you could / should use the translation `shared.pair_programming_info`:

```suggestion
= render('shared/modal', classes: 'modal-lg', id: 'modal-info-pair-programming', template: 'application/info_pair_programming', title: t('shared.pair_programming_info'))
```